src/test/java/com/linecorp/armeria/common/tracing/HttpTracingTestBase.java

```diff
@@ -8,7 +8,7 @@
 
 public abstract class HttpTracingTestBase {
 
-    public static final SpanId testSpanId = SpanId.create(1, 2, 3L);
+    public static final SpanId testSpanId = SpanId.builder().traceId(1).spanId(2).parentId(3L).build();
```

Comment:
@synk As you see here, we need to use a builder to create a `SpanId`. It also has a public constructor, but it seems to me that it's for internal use.